zipkin-lens/src/components/App/Layout.tsx

```diff
@@ -28,7 +28,6 @@ import TraceIdSearch from './TraceIdSearch';
 import TraceJsonUploader from './TraceJsonUploader';
 import { useUiConfig } from '../UiConfig';
 import { darkTheme } from '../../constants/color';
-import logoSrc from '../../img/zipkin-logo.png';
```

Comment:
this doesn't relatlvize, but since the logo will likely never change, I just moved it to public